### DISCUSSION TITLE
Scrobbler not working annymore

### DISCUSSION THREAD

**Discussion by skycamel** [rern/rAudio#543] [Bug Reports]
Since the last update (20210927) the LastFM scrobbler is not working anymore.
I get this message "Libgcrypt warning: missing initialization - please fix the application"
Anyway to fix this?

**Comment by rern** [comment]
[Libgcrypt warning: missing initialization](https://github.com/MusicPlayerDaemon/mpdscribble/issues/18)

**Comment by rern** [comment]
Update 20211112 dropped `mpdscribble` and uses a native script which runs better with options to scrobble each renderer.
